src/components/Quotes.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import '../styles/Quote.css';
+
+class Quotes extends Component {
+  render () {
+      return (
+        <section className="quotes">
+          <div className="quotes-container">
+            <div className="customer-blockquote">
```

Comment:
This `customer-blockquote` section looks like it is repeated, which makes me think it would lend itself really well to becoming a component?